README.md

```diff
@@ -5,26 +5,66 @@ Workspace is a tool to orchestrate and bring consistency to your project environ
 ## Documentation
 
 ### Getting Started
+
 #### Requirements
+
  - `PHP-7.2+`
  - `sodium` php extension installed and activated in php.ini if it's not enabled by default 
  - `docker 17.04.0+`
  - `docker-compose (compose file version 3.1+)`
+
 #### Installation
-Download the `ws` file from the [Latest Release](https://github.com/my127/workspace/releases/latest) make executable and move to a location in your PATH, eg.
+
+Download the `ws` file from the [Latest Release](https://github.com/my127/workspace/releases/latest) make executable and 
+move to a location in your PATH, eg.
+
 ```
 chmod +x ws && sudo mv ws /usr/local/bin/ws
 ```
-#### Creating a workspace
-```
-# TODO
-```
-### Anatomy of a workspace
 
 #### Key Concepts
- - [Workspace](docs/concepts/workspace.md)
- - [Harness](docs/concepts/harness.md)
+
+##### Workspace
+ 
+`workspace` describes both the CLI tool `ws` and the project environment itself. It is dependent on a harness definition.
+ 
+##### Harness
  
+ A harness is a "flavour" of workspace which defines the blueprint to which a project environment will adhere. It  
+ incorporates two distinct elements:
+ 
+ 1. *Skeleton* - a project stub applied when the workspace is first created with a given harness 
```

Comment:
These are both "harness" concepts unrelated to the workspace tool - perhaps we could document this in the harness repos and link to it from here?
e.g.
* https://github.com/inviqa/harness-base-php
* https://github.com/inviqa/harness-keycloak
* https://github.com/inviqa/harness-node
and maybe a link to https://my127.io/workspace/harnesses.json ?

Reply:
:+1: most docs are going to be harness specific, we can flesh out the basic types and concepts here and better explain its more like a task runner with templating where you have shared harnesses to give you a quick start when creating a project.

Reply:
Agreed - there's definitely value in summarising the Harness concept here and signposting to the detail